api/buildcraft/api/core/BuildCraftAPI.java

```diff
@@ -18,9 +18,17 @@
 import net.minecraftforge.fml.common.Loader;
 import net.minecraftforge.fml.common.ModContainer;
 
-public final class BuildCraftAPI {
+import buildcraft.api.template.ITemplateRegistry;
+import buildcraft.api.transport.IStripesRegistry;
+import buildcraft.api.transport.pipe.IPipeRegistry;
+import buildcraft.api.transport.pluggable.IPluggableRegistry;
 
-    public static IBCFakePlayer fakePlayerProvider;
+public final class BuildCraftAPI {
+    public static IFakePlayerProvider fakePlayerProvider;
+    public static IPipeRegistry pipeRegistry;
+    public static IPluggableRegistry pluggableRegistry;
+    public static IStripesRegistry stripeRegistry;
+    public static ITemplateRegistry templateRegistry;
```

Comment:
It's common for mods to only include the packages they need in the built jars, so we need to remove inter-package dependcies. This needs doing for the whole api (I'll do it alongsde my pass of the main code cleanup when everything is closer to completion) so for now please move this back into PipeApi. (You can leave templateRegistry there or add a TemplateApi class to put it in)